feature/collectionSheet/src/commonMain/kotlin/com/mifos/feature/individualCollectionSheet/individualCollectionSheetDetails/IndividualCollectionSheetDetailsScreen.kt

```diff
@@ -210,6 +210,8 @@ private fun IndividualCollectionSheetItem(
     modifier: Modifier = Modifier,
     onClick: () -> Unit,
 ) {
+    val loan = client.loans?.getOrNull(index)
```

Comment:
i see that this index is not use anywhere except finding the loan. so instead of passing index why not passing the loan itself from place this composable is called